README.md

```diff
@@ -17,7 +17,7 @@ See [requirements](#requirements) if you aren't sure whether you have this.
 1. Extract the files and put them in your Neovim directory
    (usually `$XDG_CONFIG_HOME/nvim/`).
 2. Execute the `:UpdateRemotePlugins` and restart Neovim.
-3. Call `deoplete#enable()` or set "let g:deoplete#enable_at_startup = 1" in
+3. `Call deoplete#enable()` or `set "let g:deoplete#enable_at_startup = 1"` in
```

Comment:
The "C" in "Call" should be made lowercase since you want it to be a copy/paste-able command:

    Add `call deoplete#enable()` or `set "let g:deoplete#enable_at_startup = 1"` in